_posts/2021-01-23-modeling-ui-state.md

```diff
@@ -0,0 +1,196 @@
+---
+layout: post
+title: Modeling UI State on Android
```

Comment:
Maybe you could rename to something like: "Modelling UI State on Android, the funcional way", or something like that.
I felt tricked into reading functional stuff, since the name did not imply anything about FP :p